### PR TITLE
Seed data

### PR DESCRIPTION
**Pull request by theo333**

### CODE REVIEW

File: public/bundle.js [hunk 1]
```diff
@@ -0,0 +1,100 @@
+/******/ (function(modules) { // webpackBootstrap
```
Comment (erin-mac) [inline public/bundle.js]: GET THIS OUT OF HERE

**Review by erin-mac — APPROVED** [review]

**Review by erin-mac — CHANGES_REQUESTED** [review]
See my Webpack comment

File: public/bundle.js (hunk 1 above)
Comment (theo333) [reply public/bundle.js]: Ok, all done.  You can approve now, pleeeease!!!!

**Review by erin-mac — APPROVED** [review]
This looks great. SHIP IT!!!

**Pull request merged**